tests/math/test_matrix_manipulation.py

```diff
@@ -176,7 +187,7 @@ def func_for_autodiff(mat):
             (1, [[[0.2, 0.5], [1.2, 1.1]], [[-0.3, -0.2], [-1.3, 1.9]], [[0.2, 0.1], [0.2, 0.7]]]),
         ],
     )
-    def test_autograd(self, i, base_matrix, tol):
+    def test_autograd(self, i, base_matrix, tol: float):
```

Comment:
Did you manually add these type hints or is this the product of a formatting tool? I dont think we need/want type hints in tests so I would remove those here.

Reply:
removed those, had added the type hints as part of an automated refactor but realized probably shouldn't be added